apps/generator-cli/src/app/services/version-manager.service.ts

```diff
@@ -96,7 +96,7 @@ export class VersionManagerService {
         this.logger.log(
           chalk.red(`Unable to query repository, because of: "${e.message}". Return default versions instead.`)
         );
-        this.printResponseError(e);
+        //this.printResponseError(e);
```

Comment:
Should this line really be commented out?

Reply:
good catch. will uncomment it in my upcoming PR

thanks for reviewing

Reply:
done via https://github.com/OpenAPITools/openapi-generator-cli/pull/831